tests/acceptance/features/apiArchiver/downloadById.feature

```diff
@@ -124,3 +124,40 @@ Feature: download multiple resources bundled into an archive
       | user-agent | archive-type |
       | Linux      | tar          |
       | Windows NT | zip          |
+
+
+  Scenario Outline: user download the folder shared with them using sharingNg API
```

Comment:
```suggestion
  Scenario Outline: download the folder shared with them
```
it sounds like the user is downloading the file using sharingNG API. Also, isn't this scenario already in the previous step? 
IMO, the given step in the above step should be refactored to sharingNG API. Therefore this scenario might be redundant.